src/utils/gaa-test.js

```diff
@@ -47,8 +47,7 @@ const PRODUCT_ID = 'scenic-2017.appspot.com:news';
 const IFRAME_URL = 'https://localhost/gsi-iframe';
 const GOOGLE_3P_AUTH_URL = 'https://fabulous-3p-authserver.glitch.me/auth';
 const CASL_URL = 'https://example-casl.com';
-const GOOGLE_API_CLIENT_ID =
-  '520465458218-e9vp957krfk2r0i4ejeh6aklqm7c25p4.apps.googleusercontent.com';
```

Comment:
I noticed this realistic Client ID was preventing me from committing changes to the file. Luckily, it doesn't need to be realistic